descriptors/ModuleDescriptor-template.json

```diff
@@ -466,6 +466,47 @@
             "circulation-storage.requests.item.delete"
           ]
         },
+        {
+          "methods": [
+            "POST"
+          ],
+          "pathPattern": "/circulation/requests/{id}/move",
+          "permissionsRequired": [
+            "circulation.requests.item.move.post"
+          ],
+          "modulePermissions": [
```

Comment:
Please create a local permission set for this, to reduce the chances of exceeding the header size, see [CIRC-352](https://issues.folio.org/browse/CIRC-352)

Reply:
Will do.

Reply:
As discussed, none of the request interface permissions have local permissions created. Therefore, agreed not to change for just this one request.